labex/src/components/AdmCreateTripPage.js

```diff
@@ -0,0 +1,133 @@
+import React, { useState } from 'react'
+import {PageStyle, ActualPage, Inputs, Button, BackButton} from  './styled'
+import axios from 'axios'
+
+import NavBar from './NavBar'
+
+    const useForm = initialValues => {
+        const [form, setForm] = useState(initialValues);
+        const onChange = (name, value) => {
+            const newForm = {...form, [name]: value}
+            setForm(newForm)
+        }
+        return {form, onChange}
+    }
+const FormPage = () => {
+
+    const {form, onChange} = useForm({name:"", planet: "", date:"", description:"", durationInDays:""})
```

Comment:
Quando um objeto possui muitas propriedades, é bastante comum quebrar uma linha para facilitar a visualização:
```suggestion
    const {form, onChange} = useForm({
       name:"",
       planet: "",
       date:"",
       description:"",
       durationInDays:""
    })
```